packages/rancher-backup-crd/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,13 +0,0 @@
---- charts-original/Chart.yaml
```

Comment:
This patch file is needed in the past because we never cut a real release for the 
upstream rancher/backup-crd chart. 
From now on, we will cut real releases in the upsteam chart, so this patch file is not needed anymore.